src/UI/processesmodel.cpp

```diff
@@ -1,14 +1,28 @@
 #include "pch.h"
 #include "processesmodel.h"
 #include "processmonitor.h"
-#include "processinfo.h"
+
+namespace Column{
```

Comment:
Use `enum class` instead of namespace and give this enum a more meaningful name like `ProcessTableColumn`.

Reply:
I changed enum name to what you suggested.